packages/nodes-base/nodes/MongoDb/MongoDb.node.ts

```diff
@@ -38,7 +38,7 @@ export class MongoDb implements INodeType {
 		name: 'mongoDb',
 		icon: 'file:mongodb.svg',
 		group: ['input'],
-		version: [1, 1.1],
+		version: [1, 1.2],
```

Comment:
What happens to version `1.1`? If you are adding a new node version for a feature that only works with that version you would need to add it to the array.

Reply:
TBH, I'm not sure if I have to update this version or not. Is it the version of the node, or some requirements ?

Assume I'm John Snow, because I know nothing about n8n 😇

Reply:
In the meantime, I've reverted this change